LDAR_Sim/src/economics/cost_mitigation.py

```diff
@@ -48,13 +48,9 @@ def cost_mitigation(simulation_dfs, ref_program, no_program, output_directory):
         'GWP_CH4': df['p_params']['economics']['GWP_CH4'],
         'carbon_price_tonnesCO2e':
         df['p_params']['economics']['carbon_price_tonnesCO2e'],
-        'social_cost_CH4_tonnes':
-        df['p_params']['economics']['social_cost_CH4_tonnes'],
         'cost_CCUS': df['p_params']['economics']['cost_CCUS'],
-        'cost_low_bleed_pneu_tCO2e':
-        df['p_params']['economics']['cost_low_bleed_pneu_tCO2e'],
         'total_program_cost': df['timeseries']['total_daily_cost'].sum()
-        } for df in simulation_dfs]
+    } for df in simulation_dfs]
```

Comment:
Why have the social cost of CH4 and cost of low bleed been removed here? (disregard if this makes perfect sense or was an offline convo)

Reply:
I'm not sure for SSC but for the low bleeds, the economics were so bad that it made the graph look terrible - so we decided to only include things with $/tCO2e < 300